client/components/common/UserProfile.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import DropDown from './DropDown';
+import profile from '../../assets/img/profile_picture.jpg';
```

Comment:
### Nitpick

This name isn't descriptive enough, you might want to try something like `profileImage` instead.